notebooks/gotcha.clj

```diff
@@ -0,0 +1,27 @@
+(ns gotcha
+  [:require
+   [scicloj.kindly.v4.kind :as k]])
+
+(k/md "# Gotchas...
+Although we try to avoid such things, sometimes, when you're fighting the host language, it's just not practical to carry over the original conventions. Here we will try to keep an up-to-date list of possible surprises (that are not easy to 'fix').
+
+**^** (`Power`) - Despite allowing a wider range of characters, we cannot use `^` in Wolframite because it is reserved for adding metadata to virtually all Clojure symbols. `(^ x 2)` will probably not do what you expect!
```

Comment:
```suggestion
**^** (`Power`) - Despite allowing a wider range of characters, we cannot use `^` in Wolframite because it is reserved for adding metadata to virtually all Clojure symbols. `(^ x 2)` will probably not do what you expect! Just use `(wl/Power ...)`
```

Reply:
Not sure how to edit your commit suggestion. There's a mistake here, in that Power is not part of core (wl/...). I would indicate that (w/Power ...) is the safe way of doing things, but that you can also make your own shortcuts, e.g. the Python-friendly (w/** ...).